executor/executor.go

```diff
@@ -296,6 +297,58 @@ type ShowDDLJobsExec struct {
 	jobs   []*model.Job
 }
 
+// ShowDDLJobQueriesExec represents a show DDL job queries executor.
+type ShowDDLJobQueriesExec struct {
+	baseExecutor
+
+	cursor int
+	jobs   []*model.Job
+	jobIDs []int64
+	query  string
+}
+
+// Open implements the Executor Open interface.
+func (e *ShowDDLJobQueriesExec) Open(ctx context.Context) error {
+	if err := e.baseExecutor.Open(ctx); err != nil {
+		return errors.Trace(err)
+	}
+	jobs, err := admin.GetDDLJobs(e.ctx.Txn())
+	if err != nil {
+		return errors.Trace(err)
+	}
+	historyJobs, err := admin.GetHistoryDDLJobs(e.ctx.Txn())
```

Comment:
I think if the user gives the job ID, they want to get the corresponding query. So I think GetHistoryDDLJobs only return 10 jobs that cannot meet the demand.

Reply:
I will write a document explaining that the usage code also adds a comment.

Reply:
Could you add a TODO here? We need to return the job that the user needs.